maat-orchestration/src/main/java/uk/gov/justice/laa/maat/orchestration/dto/AddressDTO.java

```diff
@@ -0,0 +1,192 @@
+package uk.gov.justice.laa.maat.orchestration.dto;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class AddressDTO extends GenericDTO {

```

Comment:
See my comment regarding the creation of the new DTO classes

https://github.com/ministryofjustice/laa-maat-orchestration/pull/33#issuecomment-1784931001